docs/framework/wcf/samples/transport-udp.md

```diff
@@ -103,13 +103,13 @@ this.socket.SendTo(messageBuffer.Array, messageBuffer.Offset, messageBuffer.Coun
 ```  
   
 ### The UdpChannelListener  
- The``UdpChannelListener that the sample implements derives from the <xref:System.ServiceModel.Channels.ChannelListenerBase> class. It uses a single UDP socket to receive datagrams. The `OnOpen` method receives data using the UDP socket in an asynchronous loop. The data are then converted into messages using the Message Encoding Framework.  
+ The `UdpChannelListener` that the sample implements derives from the `<xref:System.ServiceModel.Channels.ChannelListenerBase>` class. It uses a single UDP socket to receive datagrams. The `OnOpen` method receives data using the UDP socket in an asynchronous loop. The data are then converted into messages using the Message Encoding Framework.  
```

Comment:
\` and \` should be removed from \<xref:System.ServiceModel.Channgels.ChannelListenerBase>